client/native.lua

```diff
@@ -1340,6 +1340,20 @@ end
     end)
 
 --// N3d //--
+    function GetScaleformStatus()
+        local ActiveList = {}
+        local InactiveList = {}
+        for i = 1, 10 do
+            local handleName = "utility_lib_" .. i
+            if Utility.Cache.N3d[handleName] then
+                table.insert(ActiveList, {Name = handleName, Data = Utility.Cache.N3d[handleName]})

```

Comment:
Hint: put the names in camel cases, since i usually use this type of case for table keys

Reply:
I can't understand what you mean about camel cases you mean dictionary way like this as the answer?
```lua
{
    ["utility_lib_1"] = blablabla,
    ["utility_lib_2"] = blablabla,
    ["utility_lib_3"] = blablabla,
    ["utility_lib_4"] = blablabla,
    ["utility_lib_5"] = blablabla,
    ["utility_lib_6"] = blablabla,
    ["utility_lib_7"] = blablabla,
    ["utility_lib_8"] = blablabla,
    ["utility_lib_9"] = blablabla,
    ["utility_lib_10"] = blablabla,
}
```